### DISCUSSION TITLE
Can you provide list of sub accounts that are supported in Expenses?

### DISCUSSION THREAD

**Discussion by avinashkanaujiya** [ananthakumaran/paisa#55] [Q&A] [answered]
By supported I mean, the one where icon is specific to the account type like food is.

**Comment by ananthakumaran** [comment]
checkout https://github.com/ananthakumaran/paisa/blob/master/src/lib/icon.ts

**Reply by avinashkanaujiya** [reply]
can support for more be added?, can I try?

**Reply by ananthakumaran** [reply]
Please wait for some time. I don't think the current approach will scale, I am planning to allow users to select icons at the account level and ship a predefined set of icons like https://squidfunk.github.io/mkdocs-material/reference/icons-emojis This along with searchable text input in configuration page should work. But, I first need to figure out the details of icon packaging.

**Reply by ananthakumaran** [reply, marked as answer]
This is available on v0.5.5 https://paisa.fyi/reference/accounts/#icons
